### ISSUE TITLE
[Bug] Failed to load support for server version 1_21_R1

### ISSUE THREAD

**Issue by MineKing9534** [Rapha149/SignGUI#17]
### API version

2.3.6

### Server software and version

Paper 1.20.1

### Steps to reproduce

```bash
Use SignGUI in Paper 1.21
```

### Expected behaviour

Don't throw an exception

### What is actually happening?

An exception is thrown
```
java.lang.ExceptionInInitializerError: null
...
	at 20b1eef199d555e75e104fb98218b1ba983420b6.jar/dev.triumphteam.gui.guis.GuiListener.onGuiClick(GuiListener.java:102) ~[20b1eef199d555e75e104fb98218b1ba983420b6.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor56.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:1.21-113-f82bea6]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.21.jar:1.21-113-f82bea6]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[paper-1.21.jar:1.21-113-f82bea6]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.21-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:3122) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:69) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:33) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:56) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:151) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1536) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:201) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:125) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1513) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1506) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:135) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1465) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1587) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1303) ~[paper-1.21.jar:1.21-113-f82bea6]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:330) ~[paper-1.21.jar:1.21-113-f82bea6]
	at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
Caused by: java.lang.IllegalStateException: Failed to load support for server version 1_21_R1
	at Plugin.jar/de.mineking.minecraft.moderation.signgui.SignGUI.<clinit>(SignGUI.java:76) ~[Plugin.jar:?]
	... 28 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:74) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at Plugin.jar/de.mineking.minecraft.moderation.signgui.SignGUI.<clinit>(SignGUI.java:73) ~[Plugin.jar:?]
	... 28 more
Caused by: java.lang.NoClassDefFoundError: net/minecraft/network/NetworkManager
Caused by: java.lang.IllegalStateException: Failed to load support for server version 1_21_R1

	at Plugin.jar/de.mineking.minecraft.moderation.signgui.version.Wrapper1_21_R1.<init>(Wrapper1_21_R1.java:40) ~[Plugin.jar:?]
Caused by: java.lang.reflect.InvocationTargetException

Caused by: java.lang.NoClassDefFoundError: net/minecraft/network/NetworkManager

	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at Plugin.jar/de.mineking.minecraft.moderation.signgui.SignGUI.<clinit>(SignGUI.java:73) ~[Plugin.jar:?]
	... 28 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.network.NetworkManager
Caused by: java.lang.ClassNotFoundException: net.minecraft.network.NetworkManager

	at io.papermc.paper.plugin.entrypoint.classloader.PaperPluginClassLoader.loadClass(PaperPluginClassLoader.java:146) ~[paper-1.21.jar:1.21-113-f82bea6]
	at io.papermc.paper.plugin.entrypoint.classloader.PaperPluginClassLoader.loadClass(PaperPluginClassLoader.java:107) ~[paper-1.21.jar:1.21-113-f82bea6]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at Plugin.jar/de.mineking.minecraft.moderation.signgui.version.Wrapper1_21_R1.<init>(Wrapper1_21_R1.java:40) ~[Plugin.jar:?]
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at Plugin.jar/de.mineking.minecraft.moderation.signgui.SignGUI.<clinit>(SignGUI.java:73) ~[Plugin.jar:?]
	... 28 more
```

**Comment by BoBkiNN** [comment]
I have made a fork that fixes this bug in #19

**Comment by Rapha149** [comment]
The pull request is merged and the new release is published.
